config/backpack/base.php

```diff
@@ -35,8 +35,23 @@
 
     // CSS files that are loaded in all pages, using Laravel's asset() helper
     'styles' => [
-        // 'styles/example.css',
-        // 'https://some-cdn.com/example.css',
+        'https://unpkg.com/bootstrap@5.2.3/dist/css/bootstrap.min.css',
+        'https://unpkg.com/@coreui/coreui@4.2.4/dist/css/coreui.min.css',
+        'https://unpkg.com/animate.css@4.1.1/animate.compat.css',
+        'https://unpkg.com/noty@3.2.0-beta-deprecated/lib/noty.css',
+
+        'https://cdn.jsdelivr.net/npm/prismjs@1.23.0/themes/prism.css',
+        'https://coreui.io/demos/bootstrap/4.2/free/vendors/simplebar/css/simplebar.css',
+        'https://coreui.io/demos/bootstrap/4.2/free/css/vendors/simplebar.css',
+        'https://coreui.io/demos/bootstrap/4.2/free/css/style.css',
+
+        // Load the fonts separately (so that you can replace them at will):
+        ['https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;0,700;1,400&display=swap', true, [], 'style'],
+
+        'https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/css/line-awesome.min.css',
+        'https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/fonts/la-regular-400.woff2',
+        'https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/fonts/la-solid-900.woff2',
+        'https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/fonts/la-brands-400.woff2',
```

Comment:
Same as my comment below, I think we should move these to the blade files.

Reply:
Rule of thumb: if it's not easy for _us_ to switch between different themes in the demo... it won't be easy for devs to switch themes, in their projects.

Reply:
Fixed!